src/main/java/com/rst/bootcamp/model/Instructor.java

```diff
@@ -0,0 +1,124 @@
+package com.rst.bootcamp.model;
+
+import java.util.List;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Transient;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+
+@Entity
+public class Instructor {
+
+	
+	public static final int ROOT_ID = 1;
+	@Id
+	@GeneratedValue
+	@JsonIgnore
+	private int id;
+	
+	@Column(unique=true)
+	private String instructorName;
+
+
+	  @JsonIgnore
+	  @ManyToOne(cascade = { CascadeType.PERSIST})
+	  private Instructor parent;
+	  
+	  @Transient
+	  private String admin;
```

Comment:
this should be of Instructor type.

Reply:
I have the parent as the type Instructor - and admin is just a field to show.  
If I keep Admin as the type Instructor. The whole Response structure keeps on going on a never ending loop.
Still looking how can I make it possible.